### DISCUSSION TITLE
[A-L] (2024/25) Foglio 1 - Esercizio 12

### DISCUSSION THREAD

**Discussion by Rurik-D** [sapienzastudentsnetwork/algebra#473] [Esercizi - Viaggi/Cherubini]
![image](https://github.com/user-attachments/assets/4b45164e-17dd-45e8-920e-8e101df05cb3)

**Comment by simyKek** [comment]
![WhatsApp Image 2024-09-28 at 09 59 01](https://github.com/user-attachments/assets/6741270c-637c-46c7-9503-ca2393bdd9a9)
![WhatsApp Image 2024-09-28 at 09 59 01 (1)](https://github.com/user-attachments/assets/52e0944c-6418-43bb-b084-ec788c82396b)
